.travis.yml

```diff
@@ -43,7 +38,7 @@ jobs:
     - env: TEST=code-coverage
 
 before_script:
-  - git clone -q -b ros2 --depth=1 https://github.com/ros-planning/moveit_ci.git .moveit_ci
+  - git clone -q -b sync-master --depth=1 https://github.com/henningkayser/moveit_ci.git .moveit_ci
```

Comment:
@henningkayser is this supposed to point to your fork?

Reply:
You're right! I should have added a TODO to this, I'm working on getting it ready (https://github.com/ros-planning/moveit_ci/pull/118)